src/service/poc_mobile.proto

```diff
@@ -891,3 +893,48 @@ message verified_subscriber_verified_mapping_event_ingest_report_v1 {
   // Timestamp in milliseconds since unix epoch
   uint64 timestamp = 3;
 }
+
+message rle_event_v1 {
```

Comment:
what does this message represent?

Reply:
Fantastic question!

Brian helped enlighten me that this message is series of correlations that leads to the location estimate. For data reason, the most we can expose of the correlation is the `id` and `timestamp`. 

It has been renamed `rle_event_v1` -> `radio_location_correlation_v1`.

Reply:
What does a correlation event entail? Some pure identifier feels like we’re outputting data simply for the sake of outputting data. I would think a safe thing to output that would also be intrinsically useful beyond token “transparency” would be to output an estimated location (in lat/lon or hex) with an estimated accuracy score and leave the info about the subscriber that provided that correlation the opaque portion that doesn’t escape our private backends